RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -4,7 +4,39 @@
 //  Copyright © yagom academy. All rights reserved.
 // 
 
+//TODO: 함수이름
+
 import Foundation
 
-print("Hello, World!")
+gameLoop: while true {
+    print("가위(1), 바위(2), 보(3)! <종료 : 0> :", terminator: " ")
+    let computer = String(Int.random(in: 1...3))
+    let input = readLine()
+    
+    switch input {
+    case "0":
+        print("게임 종료")
+        break gameLoop
+    case "1", "2", "3":
+        if let user = input, judgeGame(user, with: computer) {
+            break gameLoop
+        }
```

Comment:
```suggestion
        if let user = input, 
           judgeGame(user, with: computer) {
            break gameLoop
        }
```
저 같은 경우 가독성을 위해 한줄의 하나의 로직만 위치하도록 작성을 하는 편입니다 😄

---

현재 judgeGame 의 결과가 true 여야 gameLoop 가 종료되고 있네요.
bool 타입으로 게임결과가 어떤 상태인지 명확하게 구분을 할 수 있을까요?